libs/vkd3d/command.c

```diff
@@ -8977,6 +8977,18 @@ static void d3d12_command_list_execute_resolve(struct d3d12_command_list *list,
         src_view_desc.image_usage = VK_IMAGE_USAGE_SAMPLED_BIT;
         src_view_desc.allowed_swizzle = true;
 
+        if (!d3d12_resource_view_format_is_compatible(dst_resource, dst_view_desc.format))
```

Comment:
`d3d12_command_list_select_resolve_path` is kind of intended to have checks like these, should probably just return `VKD3D_RESOLVE_IMAGE_PATH_UNSUPPORTED` if we end up with an edge case we can't support.

Reply:
Right ... I'll fix it.

Reply:
Fixed.